llm_studio/python_configs/text_causal_language_modeling_config.py

```diff
@@ -211,6 +212,7 @@ def __post_init__(self):
         self._possible_values["batch_size"] = (1, 256, 1)
         self._possible_values["epochs"] = (0, 10, 1)
         self._possible_values["schedule"] = Schedulers.names()
+        self._possible_values["min_learning_rate_ratio"] = (0.0, 0.1, 0.0001)
```

Comment:
As we limit it here to 0.1, but say it can go to 1.0 in the tooltip, shall we make it customizable in the settings?

Reply:
will adjust the tooltip